rpc/internal/test_results.ext.go

```diff
@@ -0,0 +1,50 @@
+package internal
+
+import (
+	"github.com/evergreen-ci/cedar/model"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/pkg/errors"
+)
+
+// Export exports TestResultsInfo to the corresponding TestResultsInfo type in
+// the model package.
+func (t TestResultsInfo) Export() model.TestResultsInfo {
+	return model.TestResultsInfo{
+		Project:     t.Project,
+		Version:     t.Version,
+		Variant:     t.Variant,
+		TaskName:    t.TaskName,
+		TaskID:      t.TaskId,
+		Execution:   int(t.Execution),
+		RequestType: t.RequestType,
+		Mainline:    t.Mainline,
+	}
+}
+
+// Export exports TestResult to the corresponding TestResult type in the model
+// package.
+func (t TestResult) Export() (model.TestResult, error) {
```

Comment:
should this return a pointer?

Reply:
No because the model functions don't take pointers